### PR TITLE
test(ivy): run more router tests in CI

### PR DESCRIPTION
**Pull request by marclaval**
With the latest changes, we are down to 29 unit tests failures in the router package (< 7%).

### CODE REVIEW

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
@mhevery @IgorMinar This PR also needs approval from the router group, according to pullapprove.

File: packages/router/test/router_preloader.spec.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ describe('RouterPreloader', () => {
       });
     });
 
-    fixmeIvy('FW-???: Cannot read property \'declarations\' of undefined') &&
+    fixmeIvy('FW-???: NgModuleRef._parent is undefined') &&
```
Comment (kara) [inline packages/router/test/router_preloader.spec.ts]: Let's actually change these to `unknown` so it's clear these still need to be investigated.

**Review by kara — CHANGES_REQUESTED** [review]

File: packages/router/test/integration.spec.ts [hunk 2]
```diff
@@ -2482,7 +2464,7 @@ describe('Integration', () => {
                  expect(canceledStatus).toEqual(false);
                })));
 
-        fixmeIvy('unknown') &&
+        fixmeIvy('assertion error') &&
```
Comment (kara) [inline packages/router/test/integration.spec.ts]: Revert?

**Review by kara — APPROVED** [review]
LGTM

**Comment by kara** [comment]
@jasonaden Need assistance because I won't have permission to approve this until https://github.com/angular/angular/pull/27337 goes in.

**Comment by IgorMinar** [comment]
@marclaval the CI is failing for this PR. can you please fix that?

**Review by IgorMinar — APPROVED** [review]

**Comment by IgorMinar** [comment]
I'm rubber-stamping this but the ci failure needs to be resolved before we merge this.

**Comment by marclaval** [comment]
@IgorMinar good to go

**Review by mhevery — APPROVED** [review]

**Comment by IgorMinar** [comment]
there was another merge conflict so I rebased this PR and force-pushed

**Pull request closed without merging**